awscli/customizations/configure/get.py

```diff
@@ -30,31 +34,44 @@ class ConfigureGetCommand(BasicCommand):
          'cli_type_name': 'string', 'positional_arg': True},
     ]
 
-    def __init__(self, session, stream=sys.stdout):
+    def __init__(self, session, stream=sys.stdout, error_stream=sys.stderr):
         super(ConfigureGetCommand, self).__init__(session)
         self._stream = stream
+        self._error_stream = error_stream
 
     def _run_main(self, args, parsed_globals):
         varname = args.varname
-        value = None
+
         if '.' not in varname:
             # get_scoped_config() returns the config variables in the config
             # file (not the logical_var names), which is what we want.
             config = self._session.get_scoped_config()
             value = config.get(varname)
         else:
             value = self._get_dotted_config_value(varname)
-        if value is not None:
+
+        LOG.debug(u'Config value retrieved: %s' % value)
+
+        if isinstance(value, six.string_types):
             self._stream.write(value)
             self._stream.write('\n')
             return 0
+        elif isinstance(value, dict):
+            # TODO: add support for this. We would need to print it off in
+            # the same format as the config file.
+            self._error_stream.write(
+                'varname must be a value name, not a section or sub-section '
```

Comment:
I think we should be including the value of varname as it is not quite obvious that varname is the positional argument name in the help command. How does something like this sound?
``'varname: %s must reference a value, not a section or sub-section' % value``